practices/images-best-practices.md

```diff
@@ -0,0 +1,41 @@
+# Best practices when dealing with images for the Web
+
+In a front-end workflow we typically deal with two kinds of images:
+
+* **Icons, logos, buttons and other graphical elements** that are part of the User Interface of the page. These are usually managed by the frontend, and so they are stored with the rest of resources (templates, CSS, JS). Like those resources, they may be shared between many different pages.
```

Comment:
Are logos always UI?
Also, maybe `the rest of the resources` or `the other resources`.